src/starkware/crypto/signature/nothing_up_my_sleeve_gen.py

```diff
@@ -26,13 +26,7 @@
 import os
 
 from math_utils import ec_double, is_quad_residue, pi_as_string, sqrt_mod
-
-# Field parameters.
-# Field prime chosen to be an arbitrary prime which is:
-# (a) large,
-# (b) has a big multiplicative subgroup of size which is a power of two,
-# (c) sparse representation for efficient modular arithmetics.
-FIELD_PRIME = 2**251 + 17 * 2**192 + 1
+from starkware.cairo.lang.cairo_constants import DEFAULT_PRIME
```

Comment:
Revert this one. I prefer not to depend on cairo_constants in a non-cairo-related file.